packages/contracts/test/voting/token-voting.ts

```diff
@@ -537,12 +560,13 @@ describe('TokenVoting', function () {
       it('can execute normally if participation and support are met', async () => {
         await advanceIntoVoteTime(startDate, endDate);
 
-        await governanceErc20Mock.mock.getPastVotes.returns(30);
         await voting.connect(signers[0]).vote(id, VoteOption.Yes, false);
-        await governanceErc20Mock.mock.getPastVotes.returns(20);
-        await voting.connect(signers[1]).vote(id, VoteOption.No, false);
-        await governanceErc20Mock.mock.getPastVotes.returns(20);
-        await voting.connect(signers[2]).vote(id, VoteOption.Abstain, false);
+        await voting.connect(signers[1]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[2]).vote(id, VoteOption.Yes, false);
+        await voting.connect(signers[3]).vote(id, VoteOption.No, false);
+        await voting.connect(signers[4]).vote(id, VoteOption.No, false);
+        await voting.connect(signers[5]).vote(id, VoteOption.Abstain, false);
+        await voting.connect(signers[6]).vote(id, VoteOption.Abstain, false);
```

Comment:
could also be optimized with a loop and `Promise.all`

Reply:
Improved in e66e59747b75c6ce535bde30be1342d74a358978